ckanext/datagovuk/templates/source/new_source_form.html

```diff
@@ -90,13 +90,13 @@
     </div>
   {% endif %}
 
-  <div class="form-group form-actions">
+  <div class="form-group form-actions u-overflow-visible">
     {% block delete_button %}
       {% if data.get('id', None) and h.check_access('harvest_source_delete', {'id': data.id}) and not data.get('state', 'none') == 'deleted' %}
         {% set locale_delete = h.dump_json({'content': _('This will flag the source as deleted but keep all its datasets and previous jobs. Are you sure you want to delete this harvest source?')}) %}
         {% set locale_clear = h.dump_json({'content': _('Warning: Apart from deleting this source, this command will remove all its datasets, as well as all previous job reports. Are you sure you want to continue?')}) %}
   <div class="dropdown btn-group">
-    <a href="#" class="btn btn-danger dropdown-toggle" data-toggle="dropdown">
+    <a href="" class="btn btn-danger dropdown-toggle" data-toggle="dropdown">
```

Comment:
As this is no longer a link, I feel like it would make more sense for this to be a `button` element instead.

Besides that, looks and works a ok.

Reply:
Fixed.

Reply:
Annoying nitpick: you missed the closing tag on line 102 😬

Reply:
Oops. Fixed.